internal/operators/mce/manifest.go

```diff
@@ -34,11 +38,20 @@ func Manifests() (openshiftManifests map[string][]byte, customManifests []byte,
 	return openshiftManifests, mceManifest, nil
 }
 
-func getSubscription() ([]byte, error) {
+func getSubscriptionChannel(openshiftVersion string) string {
+	// OCP 4.(10 + x) -> MCE 2.x
+	parsedOpenshiftVersion := common.GetParsedVersion(openshiftVersion)
+	openshiftMinorVersion, _ := strconv.Atoi(parsedOpenshiftVersion[1])
+	mceChannelMinor := openshiftMinorVersion - 10
```

Comment:
this code is problematic because now if the user installs 4.14.ec.X we need to install stable 2.3.
maybe it will be better if we will have a configuration value that we will be able to change without changing the code

Reply:
I think that we need to understand if we do need to install stable 2.3.
After the attempt to do so for LSO I'm not sure that this is the right approach